tests/ci/labels.go

```diff
@@ -7,6 +7,7 @@ import (
 // Features
 var FeatureMachinepool = Label("feature-machinepool")
 var FeatureIDP = Label("feature-idp")
+var FeatureIDPNegative = Label("feature-idp-negative")
```

Comment:
I don't think we need additional label for this

Reply:
ok, so maybe later on we will add a general "negative-tc" label? so we could run all the negative tests at once?
or should each feature run all it's cases without negative/positive consideration?